project/driver_controls/src/driver_state.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+
+#include "driver_state.h"
+#include "input_event.h"
+#include "log.h"
+
+#define MAX_FSMS 10
+
+static FSM *s_driver_fsms[MAX_FSMS];
+static InputEventCheck s_event_checks[MAX_FSMS];  // Function pointer array to FSM arbiter functions
+
+static uint8_t s_active_fsms = 0;
```

Comment:
Might want to rename this to `s_num_active_fsms` or something similar. I was going to comment before I realized this wasn't a storage array. :P